src/FileHelper.php

```diff
@@ -464,15 +467,33 @@ public static function filterPath(string $path, array $options): bool
     {
         $path = str_replace('\\', '/', $path);
 
+        if (isset($options['filter'])) {
```

Comment:
What happens if the filter is not callable?

Reply:
Added exception.